src/browser/settings.jsx

```diff
@@ -33,6 +38,12 @@ var SettingsPage = React.createClass({
       showAddTeamForm: false
     });
 
+    appLauncher.isEnabled().then(function(enabled) {
```

Comment:
I did try that yesterday, without luck. But I was thinking to go back and retry it, so thank you for getting me going in the right direction.